src/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-absolute-path */
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-unresolved */
+import { Box } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import { useState } from 'react';
+
+import Footer from '/src/components/Footer/Footer';
+import Header from '/src/components/HeaderSection/Header/Header';
+
+import ChangeView from './ChangeView/ChangeView';
+import ListOfHouses from './ListOfHouses/ListOfHouses';
+import MapHouses from './MapSide/MapSide';
+
+function HomePage() {
+  const [toggleView, setToggleView] = useState(true);
+
+  return (
+    <Grid sx={{ display: 'flex', flexDirection: 'column' }} height="100vh">
+      <Grid item xs={12} marginBottom="38px" sx={{ flexBasis: { xs: '0' } }}>
+        <Header />
+      </Grid>
+      <Box
+        component="div"
+        sx={{
+          display: { md: 'flex' },
+          maxWidth: '1750px',
+          flexBasis: '100%',
+          margin: { md: 'auto' },
+        }}
+      >
+        <Grid item xs={12} md={6} sx={{ display: { xs: toggleView ? 'block' : 'none', md: 'block' } }}>
+          <MapHouses />
+        </Grid>
+        <Grid item xs={12} md={6} sx={{ display: { xs: toggleView ? 'none' : 'block', md: 'block' } }}>
+          <ListOfHouses />
+        </Grid>
+      </Box>
+      <Grid item xs={12} marginTop="28px" sx={{ flexBasis: { xs: '0' } }}>
+        <ChangeView isToggle={toggleView} isSetToggle={setToggleView} />
```

Comment:
The names could stay the same so you could: `<ChangeView toggleView={toggleView} setToggleView={setToggleView} />`.